src/main/java/ay2021s1_cs2103_w16_3/finesse/ui/MainWindow.java

```diff
@@ -193,4 +290,46 @@ private CommandResult executeCommand(String commandText) throws CommandException
             throw e;
         }
     }
+
+    /**
+     * Changes the overview menu tab to blue while the rest becomes the same as the background color
+     */
+    private void onOverview() {
+        menuOverviewTab.setStyle("-fx-background-color: #3e7b91");
+        menuIncomeTab.setStyle("-fx-background-color: #2E2E36");
+        menuExpenseTab.setStyle("-fx-background-color: #2E2E36");
+        menuAnalyticsTab.setStyle("-fx-background-color: #2E2E36");
+    }
+
+    /**
+     * Changes the income menu tab to blue while the rest becomes the same as the background color
+     */
+    private void onIncome() {
+        menuOverviewTab.setStyle("-fx-background-color: #2E2E36");
+        menuIncomeTab.setStyle("-fx-background-color: #3e7b91");
+        menuExpenseTab.setStyle("-fx-background-color: #2E2E36");
+        menuAnalyticsTab.setStyle("-fx-background-color: #2E2E36");
+    }
+
+    /**
+     * Changes the expense menu tab to blue while the rest becomes the same as the background color
+     */
+    private void onExpense() {
+        menuOverviewTab.setStyle("-fx-background-color: #2E2E36");
+        menuIncomeTab.setStyle("-fx-background-color: #2E2E36");
+        menuExpenseTab.setStyle("-fx-background-color: #3e7b91");
+        menuAnalyticsTab.setStyle("-fx-background-color: #2E2E36");
+        menuHelpTab.setStyle("-fx-background-color: #2E2E36");
+    }
+
+    /**
+     * Changes the analytics menu tab to blue while the rest becomes the same as the background color
+     */
+    private void onAnalytics() {
+        menuOverviewTab.setStyle("-fx-background-color: #2E2E36");
+        menuIncomeTab.setStyle("-fx-background-color: #2E2E36");
+        menuExpenseTab.setStyle("-fx-background-color: #2E2E36");
+        menuAnalyticsTab.setStyle("-fx-background-color: #3e7b91");
+        menuHelpTab.setStyle("-fx-background-color: #2E2E36");
+    }
```

Comment:
If the [JavaFX TabPane](https://docs.oracle.com/javase/8/javafx/api/javafx/scene/control/TabPane.html) were used, I don't think you need to write such hacky code.

Reply:
See https://github.com/AY2021S1-CS2103T-W16-3/tp/pull/75#discussion_r501141034.